packages/docutils/lib/init.ts

```diff
@@ -141,10 +147,14 @@ export async function initPython({
 }: InitPythonOptions = {}): Promise<void> {
   pythonPath = pythonPath ?? (await findPython()) ?? NAME_PYTHON;
 
-  const args = ['-m', 'pip', 'install', '-r', REQUIREMENTS_TXT_PATH, '--break-system-packages'];
+  const args = ['-m', 'pip', 'install', '-r', REQUIREMENTS_TXT_PATH];
   if (upgrade) {
     args.push('--upgrade');
   }
+  const pipMajorVersion = await getPipMajorVersion(pythonPath);
+  if (pipMajorVersion >= PIP_MIN_VERSION_NUMBER_FOR_FLAG) {
+    args.push('--break-system-packages');
+  }
   if (dryRun) {
     dryRunLog.info('Would execute command: %s %s', pythonPath, args.join(' '));
   } else {
```

Comment:
we might simply provide `BREAK_SYSTEM_PACKAGE=1` environment variable instead, so there would be no need to compare versions

Reply:
The user would then need to check whether they need this flag. Why complicate the user flow if the code can do it automatically?

Reply:
not sure what you mean. Why would they need to check it? Could you please provide an example scenario where it would be needed?